src/components/ui/Panel.jsx

```diff
@@ -91,41 +94,41 @@ export default class Panel extends React.Component {
    */
   stopResize = event => {
     event.preventDefault();
-    clearTimeout(this.timer);
 
-    const nativeEvent = event.nativeEvent;
-    const clientY = nativeEvent.changedTouches
-      ? nativeEvent.changedTouches[0].clientY
-      : nativeEvent.clientY;
-    const sizes = [0, window.innerHeight / 2, window.innerHeight];
-    const positionIndex = getClosestIndex(sizes, clientY);
+    document.removeEventListener('mousemove', this.moveCallback);
+    document.removeEventListener('touchmove', this.moveCallback);
```

Comment:
Should we also remove the listeners in componenWillUnmount, just in case ?

Reply:
Good idea, I'll add that. Thanks!